inventory-service/src/main/java/com/petclinic/inventoryservice/datalayer/Product/Product.java

```diff
@@ -11,7 +10,6 @@
 @EqualsAndHashCode
 public class Product {
     @Id
-    private String id;
```

Comment:
Why did you take out the id?

Reply:
So my initial intention was to combine the private id(Our database one) to the public one(The one our users can access) but after further consultation I learned that this might not be the best practice for security purposes. I will fix this and thank you for pointing it out.